modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -0,0 +1,129 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/authz"
+
+	channeltypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+	host "github.com/cosmos/ibc-go/v7/modules/core/24-host"
+
+	"golang.org/x/exp/slices"
```

Comment:
Do we want to remove the dependency on `golang.org/x/exp/slices` before merging? 
We can remove from the slice the old fashioned way. cc @chatton